src/Http/Controllers/VoyagerDatabaseController.php

```diff
@@ -24,12 +24,15 @@ public function index()
     {
         Voyager::canOrFail('browse_database');
 
-        $dataTypes = Voyager::model('DataType')->select('id', 'name')->get()->pluck('id', 'name')->toArray();
+        $select = Voyager::model('DataType')->select('id', 'name', 'slug')->get()->toArray();
```

Comment:
Is `$select` used for anything other than the `collect()` call 2 lines down?  If not, you should be able to combine the 2 lines like below:

```
$dataTypes = Voyager::model('DataType')->select('id', 'name', 'slug')->get()->keyBy('name')->toArray();
```

@tnylea, you merged it.  Thoughts?

Reply:
Yeah, I think that would be great adding it to one line. You're right it would definitely make it cleaner 👍

Reply:
It is a metter of taste. I don't like to build "trains" with code. It is hard to debug in future.

Reply:
Okay, committed the change.  Thanks guys!

Reply:
I don't think this is really a "train".  The `->toArray()` and `collect()` were redundantly converting back and forth, and were therefore unnecessary